### PR TITLE
build in umd format

### PR DESCRIPTION
**Pull request by egoist**
Introducing Rollup, we don't need to compile ES6 code (since you don't use it) so no plugin is used, it just compile the `ES module` and build in umd format.

The compiled file is ignored in gitignore but will be published to npm. And it's also available via unpkg.com

### CODE REVIEW

**Comment by Okazari** [comment]
#8

File: README.md [hunk 1]
```diff
@@ -19,10 +19,12 @@ Install with npm
 npm install rythm.js
 ```
 
+CDN: https://unpkg.com/rythm.js/
+
 Import rythm into your page
 
 ```html
-<script type="text/javascript" src="../rythm.js"></script>
+<script type="text/javascript" src="/path/to/rythm.js"></script>
```
Comment (Okazari) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -19,10 +19,12 @@ Install with npm
 npm install rythm.js
 ```
 
+CDN: https://unpkg.com/rythm.js/
```
Comment (Okazari) [inline README.md]: Will soon be on CNDJS too :)

**Comment by Okazari** [comment]
I usually use webpack, but rollup looks way more simple and fit really well for this use case !

I'll probably rework my code into an ES6 module version, so people building in ES6 will be able to use it in a clean way. Do you know any solutio to easily provide UMD version + ES6 modules ? How will the UMD version behave if i add Rythm.js as an export in the src file ? Will it still be working as `new Rythm()` ?

**Comment by egoist** [comment]
>  an ES6 module version

what do you mean by es6 module version? if you mean the [es](https://github.com/rollup/rollup/wiki/JavaScript-API#format) option, then it cannot be umd format

> How will the UMD version behave if i add Rythm.js as an export in the src file

nothing would change if I understand correctly, you can check out the compiled file

**Comment by Okazari** [comment]
Sorry, i was just speculating ;) ! I'd also like to add (in the future) my src as an entry point for ES6 users (so they can do `import Rythm from 'rythmjs'`)
 
Nothing to do with your PR, works like a charm :) !

**Comment by egoist** [comment]
I think current version already support `import Rythm from 'rythmjs'`

**Comment by Okazari** [comment]
Will try ;) ! Thanks again

**Comment by Okazari** [comment]
MB, just reallised your PR added `export defaults` ... So yeah it will support import Rythm from 'rythm' !

**Comment by egoist** [comment]
hmm it will be compiled to `module.exports`, so how `import` works is depending on the bundler, like in webpack  it supports `commonjs` `umd` `ES modules`

**Pull request merged**